pkgs/applications/networking/browsers/palemoon/default.nix

```diff
@@ -5,19 +5,19 @@
 , gtk2, hunspell, icu, libevent, libjpeg, libnotify
 , libstartup_notification, libvpx, makeWrapper, mesa
 , nspr, nss, pango, perl, python, libpulseaudio, sqlite
-, unzip, xlibs, which, yasm, zip, zlib
+, unzip, xlibs, which, yasm, zip, zlib, gcc
```

Comment:
Please make this a separate commit.

Reply:
indeed, this does not belong there, must be a merge issue since I didn't modify this myself -- will check